composure.sh

```diff
@@ -175,15 +175,26 @@ _transcribe ()
 
 _typeset_functions ()
 {
+  # unfortunately, there does not seem to be a easy, portable way to list just the
+  # names of the defined shell functions...
+
+  case "$(_shell)" in
+    sh|bash)
+      typeset -F | awk '{print $3}'
+      ;;
+    *)
+      # trim everything following '()' in ksh/zsh
+      typeset +f | sed 's/().*$//'
+      ;;
+  esac
+}
+
+_typeset_functions_about ()
+{
   typeset f
-  for f in "$(_get_composure_dir)"/*.inc; do
-    # Without nullglob, we'll get back the glob
-    [[ -f "$f" ]] || continue
-
-    f="${f##*/}"
-    f="${f%.inc}"
-    echo "$f"
-  done | cat - <(echo "cite\ndraft\nglossary\nmetafor\nreference\nrevise\nwrite") | sort | uniq
+  for f in $(_typeset_functions); do
+    typeset -f "$f" | grep -qE "^about | about " && echo "$f"
```

Comment:
zsh's typeset -f seems to format using tab character. im going to create an additional commit.